src/Compiler/Service/BackgroundCompiler.fs

```diff
@@ -264,6 +264,7 @@ type internal BackgroundCompiler
         enableBackgroundItemKeyStoreAndSemanticClassification,
         enablePartialTypeChecking,
         parallelReferenceResolution,
+        reuseTypecheckingResults,
```

Comment:
Backgroundcompiler already does reuse typechecking results, the only thing it does not do know is loading them in case of cold start.
The name should reflect that.

Reply:
What name would you suggest instead?

`ReuseTypecheckingResultsForColdStart` sounds overlong and also not very forward-looking, since the feature will become more capable over time.

Reply:
In this phase, when it still does not do anything, I would not even put it in as as parameter into IDE services.
(since consumers of the API will need to update the code, only to eventually figure out it does nothing atm)

It's unlikely to provide more then cold-start benefit for tooling, because all that it attempts to be doing eventually is already done in transparent compiler.

Reply:
Hm alrighty, removing things from IDE services for now.